test/old_roman_test.js

```diff
@@ -0,0 +1,32 @@
+const assert = require('chai').assert;
+const convertToOldRoman = require('../lib/old_roman');
+
+describe('Old Roman Numeral', function() {
+  it('can convert 4', function() {
```

Comment:
would love to see some tests when we are not converting to just a single digit roman numeral i.e. 44, 999, etc